docs/Scalar_Functions.md

```diff
@@ -0,0 +1,13 @@
+# Scalar functions
+
+Create new scalar functions in Go. 
+
+## Required guidelines 
+
+Please help us make the contribution process easier by providing feedback about your experience and any technical hurdles you encountered here. 
+
+### **Pure Standalone Scalar Functions Guidelines**
+- Scalar functions must conform to examples in [math](../stdlib/math) or [strings](..s/stdlib/strings).
+- You must submit a unit test in the same folder as the new scalar function implementation. 
+- You must submit an end-to-end test in [testdata](../stdlib/testing/testdata). Please look at [End_to_End_Testing.md](/End_to_End_Testing.md)for details.
```

Comment:
this link doesn't  work as said above. I would use `stdlib/strings/length_test.flux`.

That explains all because the user cannot directly test the function, but it must use it inside of a `map` transformation, for example.